README.md

```diff
@@ -42,56 +34,50 @@ Libraries: xcms, stringr, dplyr, Rcpp, openxlsx, reshape2, loder, ggplot2, gridE
   - `/tools`
 - In `/development`, clone the dev branch of the DIMS repo. 
 ```
-git clone -b dev --single-branch git@github.com:UMCUGenetics/DIMS.git
+git clone -b develop --single-branch git@github.com:UMCUGenetics/DIMS.git
+cd DIMS
+git submodule update --init --recursive
 ```
 - In `/production`, clone the master branch of the DIMS repo.
 ```
 git clone -b master --single-branch git@github.com:UMCUGenetics/DIMS.git
+cd DIMS
+git submodule update --init --recursive
 ```
 - In `/tools`, install [mono](https://www.mono-project.com/) with [GUIX](https://guix.gnu.org/) under /mono
 - In `/tools`, place the latest tested release of [ThermoRawFileParser](https://github.com/compomics/ThermoRawFileParser/releases/tag/v1.1.11) (v1.1.11) under /ThermoRawFileParser_1.1.11
 - In `/tools`, put the required Human Metabolome Database (HMDB) .RData files under /db.
 
 
 ## Usage
-The pipeline is meant to be started with the GUI, which is an R shiny program to transfer data to the HPC and start the pipeline. To open the GUI, open GUI.Rproj in Rstudio, which should open run.R and config.R. Then click "Run App" from the run.R file. 
+The pipeline can be started with a GUI, which is an R shiny program to transfer data to the HPC and start the pipeline. The GUI access can only be used when someone has access. To get access contact the bioinformaticians. 
 
 Manually starting the pipeline is also possible.
 ```
 CMD:
-  sh run.sh -i <input path> -o <output path> [-r] [-v] [-h]
+/hpc/dbg_mz/production/DIMS/run_nextflow_dims.sh -i <input path> -o <output path> -e <email> -s <samplesheet> -n <nr_replicates> -r <resolution> -p <ppm> -z <zscore> -m <matrix> -t <standard_run> [-v] [-h]
 
 REQUIRED ARGS:
-  -i - full path input folder, e.g. /hpc/dbg_mz/raw_data/run1
-  -o - full path output folder, e.g. /hpc/dbg-mz/processed/run1
+    -i - full path input folder, eg /hpc/dbg_mz/raw_data/run1 (required)
+    -o - full path output folder, eg. /hpc/dbg_mz/processed/run1 (required)
+    -e - emailadress, eg. user@umcutrecht.nl (required)
+    -s - samplesheet, eg. sampleNames.txt (required)
+    -n - number of replicates, eg. 2 (required)
+    -r - resolution, eg. 140000 (required)
+    -p - ppm, eg. 5 (required)
+    -z - zscore, 1 for Z-score and 0 for no Z-score (required)
+    -m - matrix, eg. Plasma (required)
+    -t - standard run, yes or no (required)
```

Comment:
How should I interpret this options? What makes a run a standard run, and how would the pipeline be executed differently if this was or was not a standard run?
Also, it feels a bit odd to make this a yes/no instead of true/false but that's just me.

Reply:
A standard run uses already made HMDB parts, while a non-standard run takes a database file and cuts them up in smaller parts that are used later on in the pipeline, see [HMDBparts.R](https://github.com/UMCUGenetics/CustomModules/blob/main/DIMS/HMDBparts.R). Most DIMS runs are standard runs, only some research runs are non-standard when they use a database with labelled metabolites. I do agree that true/false is better than yes/no, however, that would take more changes in the code. This could be done in a future refactor.